libraries/AP_Mount/AP_Mount_Backend.cpp

```diff
@@ -618,8 +618,16 @@ void AP_Mount_Backend::set_rctargeting_on_rcinput_change()
     const int16_t pitch_in = (pitch_ch == nullptr) ? 0 : pitch_ch->get_radio_in();
     const int16_t yaw_in = (yaw_ch == nullptr) ? 0 : yaw_ch->get_radio_in();
 
+    if (!last_rc_input.initialised)
+    {
```

Comment:
nitpick: curly bracket should be on the same line as the condition.

Reply:
Standards are standards! (personally I deeply object to this one, but I'll suck it up).